src/main/java/com/iridium/iridiumskyblock/gui/MembersGUI.java

```diff
@@ -31,9 +31,9 @@ public MembersGUI(@NotNull Island island) {
     public void onInventoryClick(InventoryClickEvent event) {
         if (members.containsKey(event.getSlot())) {
             User user = members.get(event.getSlot());
-            if (event.getClick().equals(ClickType.RIGHT)) {
+            if (event.getClick().equals(ClickType.LEFT)) {
```

Comment:
I agree left seems left seems like it should demote and right should promote, but if your going to change this you also need to change the lore in inventories.json

Reply:
i changed it cuz that already promote = right   demot = left in inventories.(YML)

Reply:
and should add else if cause users can be click with shift or middle click